spark-jobs/src/main/scala/filodb/downsampler/Utils.scala

```diff
@@ -0,0 +1,57 @@
+package filodb.downsampler
+
+import kamon.Kamon
+
+import filodb.core.store.{AllChunkScan, ChunkSetInfoReader, ReadablePartition}
+
+object Utils {
+
+  lazy val numRawChunksSkipped = Kamon.counter("num-raw-chunks-skipped").withoutTags()
```

Comment:
FYI: this will double the count, since both the downsampler and exporter use `getChunkRangeIter`.